CONTRIBUTING.md

```diff
@@ -33,6 +33,24 @@ information on using pull requests.
 This project follows [Google's Open Source Community
 Guidelines](https://opensource.google.com/conduct/).
 
+## Git Workflow
+
+This project conforms to the [Gitflow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-wokflow).
+The repo contains two important branches: the `main` branch stores the official release history, and the `develop` branch contains
+the current working codebase.
+
+As a contributor, your workflow will be to create a temporary branch off of `develop`, add your code, and create a PR back to `develop`.
+Periodically, we will coordinate to publish release PRs to merge the latest state of `develop` back into `main`.
```

Comment:
I suggest adding git commands example and put it here to describe the entire workflow. It will make new contributors' life easier